pjsip/src/pjsip/sip_transport.c

```diff
@@ -358,7 +360,8 @@ PJ_DEF(pjsip_transport_type_e) pjsip_transport_get_type_from_name(const pj_str_t
 	}
     }
 
-    pj_assert(!"Invalid transport name");
+    /* Just return intstead of raising assertion. */
+    //pj_assert(!"Invalid transport name");
```

Comment:
- Typo "intstead".
- IMO only modify this function and leave the others (don't fix it if it ain't broke :). Note that the transport name string may be parsed from incoming SIP message (which is not controlled by programmer), while in other functions, transport type/flag params are set programmatically (under programmer's control), so assertion should be the correct way already.